targeteditor/src/main/scala/explore/targeteditor/CoordinatesForm.scala

```diff
@@ -129,15 +118,16 @@ object CoordinatesForm {
           ExploreStyles.CoordinatesForm,
           FormInputEV(
             id = "search",
-            value = stateView.zoom(State.searchTerm).withOnMod(props.onNameChange),
+            value = props.name,
             validFormat = ValidFormatInput.nonEmptyValidFormat,
             label = "Name",
             error = state.searchError.orUndefined,
             loading = props.searching.get,
             disabled = props.searching.get,
             errorClazz = ExploreStyles.InputErrorTooltip,
             errorPointing = LabelPointing.Below,
-            onTextChange = _ => $.setStateL(State.searchError)(none),
+            onTextChange =
+              v => $.setStateL(State.searchTerm)(v) >> $.setStateL(State.searchError)(none),
```

Comment:
Right, I was thinking this solution, to keep the search term independent of the props